### PR TITLE
Manage config logger

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

**Comment by vincentlepot** [comment]
I don't like JSON config files because we cannot add comments to document the configuration values, and, in docker context, we need to pass configuration through environment variables so that the same image can be used in various environments

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Create config.py file from template config.py.dist and adapt them
 
 * For requests_calls consolidation
 ```
-<path/to/spark>/bin/spark-submit  --conf spark.ui.showConsoleProgress=true --master='local[3]' manage.py -a <analyzer> -i <your_export_directory> -s <start_date> -e <end_date>
+<path/to/spark>/bin/spark-submit  --conf spark.ui.showConsoleProgress=true --master='local[3]' manage.py -a <analyzer> -i <your_export_directory> -s <start_date> -e <end_date> -c settings.json
```
Comment (nberard) [inline README.md]: can be removed

File: manage.py [hunk 2]
```diff
@@ -16,13 +16,15 @@
...
     database = Database(dbname=config.db["dbname"], user=config.db["user"],
                         password=config.db["password"], schema=config.db["schema"],
                         host=config.db['host'], port=config.db['port'],
                         insert_count=config.db['insert_count'])
 
+    logger.init_logger(config.logger.get("level", ""))
     spark_session = SparkSession.builder.appName(__file__).getOrCreate()
 
-    analyzer = args.analyzer(args.input, args.start_date, args.end_date, spark_session, database)
+    analyzer = args.analyzer(args.input, args.start_date, args.end_date, spark_context, database)
```
Comment (nberard) [inline manage.py]: need to be spark_session

**Comment by nberard** [comment]
nice thing to do would be to add runtime log with syslog handler and log it with logging.getLogger("spark-stat-analyzer-runtime") or something, see https://github.com/CanalTP/spark-stat-analyzer/issues/19

File: README.md (hunk 1 above)
Comment (azime) [inline README.md]: ok

File: manage.py (hunk 2 above)
Comment (azime) [inline manage.py]: ok

**Comment by azime** [comment]
runtime: 
[2017-02-21 17:00:18,203] [DEBUG] [16855] [spark-stat-analyzer] [OK] [2017-02-21 17:00:17] [2017-02-21 17:00:13] [RequestCallsUpdater] [4]

File: analyzers/analyzer.py [hunk 3]
```diff
@@ -31,14 +32,14 @@ def launch(self):
         pass
 
     def get_log_analyzer_stats(self, current_datetime):
-        return "[spark-stat-analyzer] [OK] [%s] [%s] [%s] [%d]" %(current_datetime.strftime("%Y-%m-%d %H:%M:%S"),
-                                                                  self.created_at.strftime("%Y-%m-%d %H:%M:%S"),
-                                                                  self.analyzer_name,
-                                                                  math.floor((current_datetime - self.created_at).total_seconds()))
+        return "[OK] [%s] [%s] [%s] [%d]" %(current_datetime.strftime("%Y-%m-%d %H:%M:%S"),
```
Comment (nberard) [inline analyzers/analyzer.py]: why did u remove spark-stat-analyzer? ELK wont work with this format

**Comment by nberard** [comment]
strange, u dont have commit 83265f6cc1f75f3141115747977047f6aeb7fc2c from master...

**Comment by nberard** [comment]
could u squash your commits?

File: analyzers/analyzer.py (hunk 3 above)
Comment (azime) [inline analyzers/analyzer.py]: "spark-stat-analyzer" is name of logger, specified in formatter: logger.py line 17

Comment (nberard) [inline analyzers/analyzer.py]: oh my bad didnt see above

**Pull request merged**